charts/zitadel/templates/deployment_zitadel.yaml

```diff
@@ -22,6 +22,9 @@ spec:
         checksum/configmap: {{ include (print $.Template.BasePath "/configmap_zitadel.yaml") . | sha256sum }}
         checksum/secret-db-ssl-ca-crt: {{ include (print $.Template.BasePath "/secret_db-ssl-ca-crt.yaml") . | sha256sum }}
         checksum/secret-zitadel-secrets: {{ include (print $.Template.BasePath "/secret_zitadel-secrets.yaml") . | sha256sum }}
+        prometheus.io/path: "/debug/metrics"
+        prometheus.io/port: "8080"
+        prometheus.io/scrape: "true"
```

Comment:
Could it be helpful in some cases to have them as removable defaults in `.Values.podAnnotations` instead, so scraping can be disabled? At least the `prometheus.io/scrape` annotation?

Reply:
Fixed.